Bylaws.md

```diff
@@ -91,34 +98,36 @@ This section applies to any ballot that proposes a Final Guideline or a Final Ma
 
  2. As described in Section 2.3(3), there will be a discussion period of at least seven (7) days before votes are cast on a Draft Guideline Ballot, with the start date of such discussion period clearly specified in the ballot. The discussion period shall end and the voting period shall commence also according to the procedure specified in Section 2.3(3).
 
- 3. As described in Section 2.3(4), upon commencement of the voting period, Voting Members shall have exactly seven (7) calendar days to vote on a Draft Guideline Ballot, with the deadline clearly communicated in the ballot sent via a Public Mail List. For ballots related to the Forum level, votes must be sent to the Public Mail List of the Forum. For ballots related to a CWG, votes must be sent to the Public Mail List of the CWG. All voting will take place via the proper Public Mail List. Votes not submitted to the Public Mail List or not submitted by a designated Voting Representative, will not be considered valid, and will not be counted for any purpose. The Chair may send an email to a Public Mail List reminding Voting Members of when the voting period opens and closes.
+ 3. As described in Section 2.3(4), upon commencement of the voting period, Voting Members shall have exactly seven (7) calendar days to vote on a Draft Guideline Ballot, with the deadline clearly communicated in the ballot sent via a Public Mail List. For ballots related to the Forum level, votes must be sent to the Public Mail List of the Forum. For ballots related to a CWG, votes must be sent to the Public Mail List of the CWG. All voting will take place via the proper Public Mail List. Votes not submitted to the Public Mail List or not submitted by a Voting Representative, will not be considered valid, and will not be counted for any purpose. The Chair may send an email to a Public Mail List reminding Voting Members of when the voting period opens and closes.
```

Comment:
Should a Voting Representative from the “proposing” organization cast a vote?

Reply:
Yes, I don't think this has ever been challenged. Are you suggesting to change this language to forbid the proposing organization from casting a vote?

Reply:
Sorry, I should have added more context. I was not intending to suggest a change to forbid the action, but lacked the historical context to know whether or not the action was forbidden or permitted. Based on the last few ballots it appears it is permitted, but not required (i.e., someone from an organization can propose a ballot and not cast a vote). 

Upon voting for a ballot proposed by our team the first time, we had to seek clarification related to this from other Forum members. Making it clear that the proposing organization can also vote may help future Forum members.

Reply:
I added the following text to clarify:

`The proposing and endorsing organizations may also vote.`

That should make things clearer but please feel free to suggest a better sentence.

Reply:
LGTM